lib/install/install.go

```diff
@@ -251,11 +251,7 @@ func (i *Installer) maybeStartAgent() error {
 func (i *Installer) execute(req *installpb.ExecuteRequest) (dispatcher.Status, error) {
 	i.WithField("req", req).Info("Execute.")
 	existingOperation, _ := ops.GetWizardOperation(i.config.Operator)
-	if req.HasSpecificPhase() || existingOperation != nil {
```

Comment:
This was a crutch to be able to tell between full runs of the operation vs resuming at a later stage which I wanted to abolish.